### DISCUSSION TITLE
Run workflow manually with variables

### DISCUSSION THREAD

**Discussion by malimccalla** [community/community#26961] [Actions]
<p>I would love to be able to run my workflow manually from github with the ability to set env vars as I do so. This would be great for deploying as I could do it manually from github and add in variables like the version number. Is this feature currently avaialbe or is there any plan to add it?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/malimccalla">@malimccalla</a> ,</p>
<p>It’s recommended to track this feature on existed ticket: <a href="https://github.community/t/github-actions-manual-trigger-approvals/16233">https://github.community/t5/GitHub-Actions/GitHub-Actions-Manual-Trigger-Approvals/td-p/31504</a>. </p>
<p>You can raise a ticket <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">here</a> to confirm if there’s any progress about the feature, it will raise the feature priority, and github product manager will help to answer.</p>
<p>Meanwhile, as a workaround, you can send <a href="https://developer.github.com/v3/repos/#create-a-repository-dispatch-event" rel="nofollow noopener">Repository Dispatch Event</a> to your repository to trigger the workflow manually, you can add the version number in the event payload, and get it from ${{ github.event.client_payload.vernumber }} in the workflow.</p>
<p>My workflow:</p>
<p><a href="https://github.com/weide-zhou/ticket17/actions/runs/69528670/workflow" rel="nofollow noopener">https://github.com/weide-zhou/ticket17/actions/runs/69528670/workflow</a></p>
<p>Thanks.</p>
